taxi_service/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
+from taxi.views import index
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("taxi/", include("taxi.urls", namespace="taxi")),
```

Comment:
Ur path should redirect to the main page. Better just use “” instead of “taxi/”

Reply:
done 🍏